docs/Feeds.md

```diff
@@ -1633,6 +1633,41 @@ id: netlab360-magnitude-ek-parser
 **Notes:** This feed lists FQDN and possibly the URL used by Magnitude Exploit Kit.  Information also includes the IP address used for the domain and last time seen.
 reference: http://data.netlab.360.com/ek
 
+
+## Mirai Scanner Feed
+
+**Status:** Active
+
+### Collector Bot
+
+**Bot Name:** Generic URL Fetcher
+
+**Bot Module:** intelmq.bots.collectors.http.collector_http
+
+**Configuration Parameters:**
+```
+id: netlab360-mirai-scanner-collector
+provider: Netlab 360
+feed: Netlab 360 Mirai Scanner List
+rate_limit: FIXME
```

Comment:
I think a daily update is ok. Their last update was in February tough: http://data.netlab.360.com/mirai-scanner/